build/ci.go

```diff
@@ -322,6 +325,24 @@ func doTest(cmdline []string) {
 	build.MustRun(gotest)
 }
 
+func downloadSpecTestFixtures(cachedir string) string {
+	// Download the execution-spec-tests
+	const version = "0.2.4"
+
+	csdb := build.MustLoadChecksums("build/checksums.txt")
+	ext := ".tar.gz"
+	base := "fixtures" // TODO(MariusVanDerWijden) rename once the version becomes part of the filename
+	url := fmt.Sprintf("https://github.com/ethereum/execution-spec-tests/releases/download/v%s/%s%s", version, base, ext)
+	archivePath := filepath.Join(cachedir, base+ext)
```

Comment:
If you take the `version` as a parameter, e.g. `go run ./build/ci.go -test --test.spec "0.2.4"`. Then we don't have to muck around with making version part of the filename. In order to update the spec, we just bump the version and update the hash. 

And then this PR would be good to go, without any TODO's left..(?)

Reply:
Actually, I guess it is already good to go. TO update for a new release of the tests, we have to update the `const version` string and the hash. Doing it the way I proposed doesn't simplify anything